module/merintr/stats.c

```diff
@@ -16,13 +16,13 @@
 #include "client.h"
 #include "merintr.h"
 
-HWND hStats;                        // Window containing stats area
-static list_type stats = NULL;      // List of stats currently displayed
+HWND hStats;                        // Window containing stats area.
+static list_type stats = NULL;      // List of stats currently processed.
```

Comment:
I don't understand the change to this file.  Was "displayed" incorrect?  What does "processed" mean?